syslog/modules/lpeg/sendmail.lua

```diff
@@ -0,0 +1,32 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Sendmail module
+
+## Variables
+### LPEG Grammars
+* `grammar`
+--]]
+
+local l = require "lpeg"
+l.locale(l) -- we want l.alpha etc...
+
+local rawset = rawset
+
+local M = {}
+setfenv(1, M) -- Remove external access to contain everything in the module
+
+local space = l.space^0
+local key = l.C(l.alpha^1) * space
+local value = l.C((l.P(1) - l.P', ')^1) * space
```

Comment:
space can be removed (it can never match). 

Based on the spec lines (21-26) could be simplified down to the following:
```lua
local sep   = l.P', '
local key   = l.C(l.alpha^1)
local value = l.C((l.P(1) - sep)^1)
local pair  = l.Cg(key * '=' * value) * sep^-1
```
If you want to be more cautious about extra white space 
```lua
local sep = l.P',' * l.space^1
local key = l.C(l.alpha^1) * l.space^0
```

Reply:
I've ditched the spaces. I'm sorry for the bad copy paste behavior.